playbooks/adhoc/zabbix_setup/create_template.yml

```diff
@@ -32,8 +32,8 @@
     server: "{{ ctp_zserver }}"
     user: "{{ ctp_zuser }}"
     password: "{{ ctp_zpassword }}"
-    name: "{{ item.name }}"
     key: "{{ item.key }}"
+    name: "{{ item.name | default(item.key) }}"
```

Comment:
consider passing 'true' as the second param of default so that empty and none get converted to the default value as well.

So, something like this (untested):
```
name: "{{ item.name | default(item.key, true) }}"
```